streams/src/test/java/org/apache/kafka/streams/processor/internals/TaskManagerTest.java

```diff
@@ -336,6 +340,7 @@ public void shouldComputeOffsetSumForUnassignedTaskWeCanLock() throws Exception
 
         replay(stateDirectory);
         taskManager.handleRebalanceStart(singleton("topic"));
+        taskManager.addTask(new StateMachineTask(taskId00, Collections.emptySet(), true));
```

Comment:
We have stricter check if a tasks exists now, so we need to add a task to avoid that `getTaskOffsetSums` fails with "unknown task exception"

Reply:
Doesn't this invalidate the test though? We're specifically testing whether we can compute offset sums for tasks that we _don't_ own

Reply:
Oh. Good point. I guess I did not pay enough attention to what this test actually verifies... _urgs_